docs/index.md

```diff
@@ -1,10 +1,63 @@
-# ESSdiffraction
+:::{image} _static/logo.svg
+:class: only-light
+:alt: ESSdiffraction
+:width: 60%
+:align: center
+:::
+:::{image} _static/logo-dark.svg
+:class: only-dark
+:alt: ESSdiffraction
+:width: 60%
+:align: center
+:::
 
-<span style="font-size:1.2em;font-style:italic;color:#5a5a5a">
+```{raw} html
+   <style>
+    .transparent {display: none; visibility: hidden;}
+    .transparent + a.headerlink {display: none; visibility: hidden;}
+   </style>
+```
+
+```{role} transparent
+```
+
+# {transparent}`ESSdiffraction`
```

Comment:
This creates an invisible top level heading. It is invisible because the logo acts as the heading. But it is still good to have so that the `##` headings below are actually second level and don't get promoted to `#` heading.

I used the same implementation in Scitacean and it works well.